app/views/bulk_actions/_checksum_report_job.html.erb

```diff
@@ -0,0 +1,3 @@
+<% if bulk_action.present? && bulk_action.status == 'Completed' && File.exists?(File.join(bulk_action.output_directory, Settings.CHECKSUM_REPORT_JOB.CSV_FILENAME)) %>
```

Comment:
Can this conditional be moved to a helper or controller method?  Prefer to keep code in templates to a minimum.

Reply:
I think the safe navigation operator could be used here too (e.g. `bulk_action&.status`) allowing you to drop the first clause